### PR TITLE
chore: add the ability to specify a default member

### PR DESCRIPTION
**Pull request by f01dab1e**
# Description

add the ability to specify a default member

## Problem

step towards https://github.com/noir-lang/noir/issues/794

# PR Checklist

- [x] I have tested the changes locally.
- [x] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: crates/nargo/src/manifest/mod.rs [hunk 1]
```diff
@@ -53,8 +53,10 @@ pub struct Workspace {
 }
 
 #[derive(Default, Debug, Deserialize, Clone)]
+#[serde(rename_all = "kebab-case")]
 pub struct WorkspaceConfig {
     pub members: Vec<String>,
+    pub default_member: Option<String>,
```
Comment (kevaundray) [inline crates/nargo/src/manifest/mod.rs:59]: can you add a comment to specify what a default member is?

File: crates/nargo/src/manifest/mod.rs [hunk 2]
```diff
@@ -53,8 +53,12 @@ pub struct Workspace {
 }
 
 #[derive(Default, Debug, Deserialize, Clone)]
+#[serde(rename_all = "kebab-case")]
 pub struct WorkspaceConfig {
+    /// List of members in this workspace.
     pub members: Vec<String>,
+    /// Specifies the default crate to interact with in the context (similarly to how we have nargo as the default crate in this repository).
+    pub default_member: Option<String>,
```
Comment (f01dab1e) [inline crates/nargo/src/manifest/mod.rs:61]: maybe we should use `PathBuf` instead of `String` here

Comment (f01dab1e) [inline crates/nargo/src/manifest/mod.rs:61]: maybe we should use PathBuf instead of String here

Comment (kevaundray) [reply crates/nargo/src/manifest/mod.rs:61]: Good point -- pathBuf would better convey that you could do "crates/folder/member"

**Review by kevaundray — APPROVED** [review]
LGTM -- good point about using PathBuf

**Review by kevaundray — APPROVED** [review]

**Pull request merged**